test/tests/parallel/set1/spam_virus/rspamd_full.bats

```diff
@@ -214,7 +221,7 @@ function teardown_file() { _default_teardown ; }
   _service_log_should_contain_string 'rspamd' 'S \(add header\)'
   _service_log_should_contain_string 'rspamd' 'add header "Gtube pattern"'
 
-  _print_mail_log_for_id "${MAIL_ID4}"
+  _print_mail_log_for_id "${MAIL_ID_HEADER}"
```

Comment:
As per context feedback on `_print_mail_log_for_id "${MAIL_ID_SPAM}"`, I didn't observe any Dovecot logs related to storing mail into junk.

Since this mail appeared identical behind being sent with `--unchecked`, curious why the test passes?

Reply:
The spam mail is rejected AFAIK; naturally, you would not observe Dovecot storing anything at all, right?

More context: the mail that has the id `MAIL_ID_SPAM` may appear misleading to you (and to others as well); referring to [the Rspamd docs for GTUBE again](https://rspamd.com/doc/gtube_patterns.html), you can see that the GTUBE pattern for `MAIL_ID_SPAM` is a pattern that results in a `reject`, while `MAIL_ID_HEADER` just adds a header. I will rename the mail IDs, so this confusion is out of the world.